data/SrimanNarayana7.json

```diff
@@ -0,0 +1,19 @@
+{
+"name": "SrimanNarayana",
+"displayStatsPublic": true,
```

Comment:
We've recently removed the public stats from LinkFree, so now no longer need `displayStatsPublic` key. 
So please update accordingly. 
```suggestion
```